docs/overview/release-lifecycle.md

```diff
@@ -43,6 +43,8 @@ server repository][zulip-server].
   upgrading to a new major release series, We recommend always
   upgrading to the latest maintenance release in that series, so that
   you use the latest version of the upgrade code.
+- For the dates of past stable releases, see [the Zulip blog][blog],
+  or specifically the [list of major releases][blog-major-releases].
```

Comment:
Hmm, I feel like "stable releases" is talking about not just major releases, but also minor ones; and of course that information is also available in the changelog, so I'm a little uncertain what we're doing with this sentence. 

Would it make sense to instead make "stable releases", above, a link?

Reply:
My goal in this sentence is to give the reader a way to quickly determine which releases are 18 months old; which releases are soon to be 18 months old; when the release they're currently using will turn 18 months old; etc.

We could link to the changelog https://zulip.readthedocs.io/en/latest/overview/changelog.html instead. That'd make it possible to answer those questions, but it's pretty cumbersome — you have to scroll past a *lot* of release-notes details if what you're looking for is the versions released around a certain date.

I'm not sure I understand the first bit. This section of the page defines "stable release" to include both major and minor releases. The link is to a page that includes both kinds of stable release, with dates:
![image](https://github.com/zulip/zulip/assets/28173/63d76b45-d7ca-4816-bf16-449509521ccd)

Reply:
Well the `blog-major-releases` does not include maintenance updates; I worry about it being confusing. I think we should just link to https://blog.zulip.com/tag/release-announcements/ here, i.e. all release announcements?

Reply:
Sure, WFM. I'll revise.